src/main/java/org/craftercms/studio/impl/v1/repository/git/StrSubstitutorVisitor.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2007-2019 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.craftercms.studio.impl.v1.repository.git;
```

Comment:
Seems like class does not belong to `git` package. It is not using git operations (plain file system).

Reply:
I'm not sure where to move it, `org.craftercms.studio.impl.v1.repository` or something like `org.craftercms.studio.impl.v1.util`?

Reply:
`util` if it is useful for other services, otherwise `repository`